gadgets/src/integers/uint/tests/uint128.rs

```diff
@@ -225,8 +226,8 @@ fn test_uint128_sub_constants() {
     for _ in 0..1000 {
         let mut cs = TestConstraintSystem::<Fr>::new();
 
-        let a: u128 = rng.gen_range(u128::max_value() / 2u128..u128::max_value());
-        let b: u128 = rng.gen_range(0u128..u128::max_value() / 2u128);
+        let a: u128 = rng.gen_range(u128::MAX / 2u128..u128::MAX);
```

Comment:
Explain: This change is due to a suggestion by Rust, as `max_value()` is no longer the recommended way.